packages/material-ui/src/ListItem/ListItem.js

```diff
@@ -18,21 +18,19 @@ export const styles = theme => ({
     width: '100%',
     boxSizing: 'border-box',
     textAlign: 'left',
-    paddingTop: 11, // To use 10px in v4.0.0
-    paddingBottom: 11, // To use 10px in v4.0.0
-    '&$selected, &$selected:hover': {
+    paddingTop: 11, // To use 10px in v4
+    paddingBottom: 11, // To use 10px in v4
+    '&$selected, &$selected:hover, &$selected:focus': {
       backgroundColor: theme.palette.action.selected,
     },
   },
   /* Styles applied to the `container` element if `children` includes `ListItemSecondaryAction`. */
   container: {
     position: 'relative',
   },
-  // TODO: Sanity check this - why is focusVisibleClassName prop apparently applied to a div?
+  // To remove in v4
   /* Styles applied to the `component`'s `focusVisibleClassName` property if `button={true}`. */
-  focusVisible: {
-    backgroundColor: theme.palette.action.hover,
```

Comment:
This style is almost 3 years old: https://github.com/mui-org/material-ui/commit/9a9ea7e1ffff446e766dbf27bc2153154a83a2db.

Reply:
Gotcha. The changes look great! And all tests passing 👍